tutorials/Basics/Methods-and-Parameters-for-Linking.ipynb

```diff
@@ -0,0 +1,1455 @@
OmShoZmMRERERkf55nbvvTh4ws68BtwJvAl7QRh7fdfcDfSjbmlCDVrqWC4wHbSmwfTTPTLVBIwTDmSgGjBYMM8sMMxYRERER2SgGEXKcbszGxw6Y2U+BY9e8QAOgBu0AZT30WWNq2x1P2Wrsadb40uR42/QY2uRSOO7OKLBjhbRJ6cZtu2NBs8bXpvezltTJGhfbammgdpfm6dWyPZ0s6dPJh6fG2IqIiIj0Rw++Z20xswNZCdx9qo1y7AROBi5p8743mNkRwG7gy8DbmjWU1ys1aEVERERERDYAi1rVFwAB8JEWyW8G3gpcRzRu9olE429PNbPHuvv+fpa1V9SgFRERERERGbzpdnpgW/gwcDpwtrvfkJXQ3S9OHbrSzK4Fvga8DnjfKsuyJtSgHaB2wxLSYaztLn+TDv9N55M8nxWOnM4nef+sc2m9CjlO7/diSZ1W4dm9CCvuJOS4VxRiLCIiItJ/Zraq7129+M5mZu8H3gD8nrtf2E0e7n6Fmd1DtAzQUNCaKiIiIiIiIkPMzN5DFD58nrt/bJXZBcDKE+OsM+qhFRERERERWaVBRcaZ2buAdwDvcPcPrzKvZxGtZdvO+rXrghq0A9TtQ591XTKst1VIa1bIbTKUOB0qnDWTcda5TnQSqtuPGYj7EVbcq5mLFUYsIiIiIgBm9gbgfKLZif/VzJ6QOF1x9+vidFcDT3V3S1x7HfAZ4EagBvwa8EbgZ8BfrEX5e0ENWhERERERkeF0WvzzOfGW9HPghIxrfwK8FtgFFIA7gE8A73X3Az0tZR+pQSsiIiIiIrJKg4iic/dTuk3n7i/pdXkGQQ3aAWo3rDQd8pvcz5oBudWswt3mkxVW3Mksx1ll6yTkthchv1khxv26f6t82z3XTToRERERkY1ADVoREREREZFVUsfCYGjZHhERERERERlKatCKiIiIiIjIUFLI8QD1Oywha8xsej/rXFp6vGm713Wrk7Gn/Vo2p92xsINYmkfhLSIiIiKDp+9kg6EeWhERERERERlK6qEVERERERFZJfXQDoYatAPUi4c+K6y4Vfhv1tJAK+XZyble6ddyN90uqdOvMGItzSMiIiIi0hmFHIuIiIiIiMhQUg+tiIiIiIjIKimKbjDUoF0nskKHu/3H0cksx726f69CkHv1mrs9t95mJ9YHpIiIiIjI4RRyLCIiIiIiIkNJDVoREREREREZSgo5HqCsWYbXIsS02/tnhRWvdblXk3YtQof7ETotIiIiIuuPvr8NhnpoRUREREREZCiph1ZERERERGSV1EM7GOqhFRERERERkaGkHtp1opMxq/3460+rJX7avf96XrZnLdKu5r3RX/VERERERDqjBq2IiIiIiMgqqXNiMBRyLCIiIiIiIkNJPbRDoFfhwN3es5Mw4kH/ZapfIceDvE5ERERE1j991xsM9dCKiIiIiIjIUFKDVkRERERERIaSQo4HqNvZgtsNZ1jN7MjJa4cpfKIfIdfrIR8RERERWd/0vW8w1EMrIiIiIiIiQ0k9tCIiIiIiIqukHtrBUA9tm8xs3Mw+Zmb3mNmCmX3XzJ476HKJiIiIiIhsVmrQtu9S4GXA24HfBK4HLjWzZ/fjZmZ2aOtFHs221Vy7XrdevaZe1bOIiIiISD+tpuPNzB5iZl8ws2kzmzGzy83spH6XuZcUctyGuNH6DOAMd780PnYVcCLwUeDyARZPREREREQGbICdGZcCvwScB9wKnEXU8Xaau6/YTjGzI4BrgN3AK4A6Uefd183sMe5+Z78L3gtq0Lbn+cA0cNnSAXd3M7sIuMDMTnL36wdWOhERERER2XRW2fH2RmAr8Dh3vzu+9ltEjeK3Aef2seg9owZte04Grnf3MHX8B8nzSwfN7ECL/LZ0cvNul/dZTb7tWs3927UeQnfXQxlEREREZP0a0PfF1XS8PR+4YqkxG1+718y+BJzBkDRoNYa2PduBfU2O70ucFxERERERWUvtdLwdxszKwEOAHzU5/QPgiDgked1TD237sroil51z96msjMwsnJ6etqmpzGQiIiIiIpvO9PQ0wOSgy9GJ6elpVvPdPn7NW1pFejZpZ2wHbmqStFXH21bAaN1ptzurPOuBGrTt2Uvzh2Fb/LPZg5AlBILp6emluITpbgsmbVkK8VY994/quP9Ux/2nOl4bquf+Ux33n+q4vyaJvi8Pi2k41ChdjW4b8W13vPX42nVBDdr2/Bh4gZkFqe78R8c/m3XVr8jd8/DAWNtWPbqyOqrn/lMd95/quP9Ux2tD9dx/quP+Ux1LkrsfP8Dbd9vxtp+owdrLTruB0Bja9lwKTAGnpY6fCdyoGY5FRERERGQAfgw80szS7brMjjd3XwBuofkY20cDe9x93Ycbgxq07bocuAr4pJmdY2ZPM7MLgScBbxpoyUREREREZLNaTcfbpcAzzeyopQNmti3O6/M9LmffKOS4DfHU16cDH4i3KaJles5w9y8NsGgiIiIiIrJ5JTvethOtIfsKoo635y0lMrOrgae6e3JtoY8ALwcuN7N3A3Xg7fHPD6xJ6XtADdo2uftB4PXxJiIiIiIiMlCr6Xhz9/vM7MlEDduLiaJ3rwGe4u6397PcvWTuQzF51YakCQXWhuq5/1TH/ac67j/V8dpQPfef6rj/VMci64fG0IqIiIiIiMhQUoNWREREREREhpJCjkVERERERGQoqYdWREREREREhpIatCIiIiIiIjKU1KAVERERERGRoaQG7QCY2biZfczM7jGzBTP7rpk9d9DlGhQzO9XMLjSzG81s3szuNLPPm9mjU+muNjNvsv1DkzzbrmMze6aZXRun221mf21mU6vJc70xs1NWqDs3s0ek0va8PjZJHV+YUcduZkfF6fQct8nMjjWzPzOzb5rZbFxPp6yQdiie23bzXCvt1LGZTZrZ283s62Z2X5zuv8zsD8ysmErb9mdNnF51/EC6oflsGMY6NrMTMp5NN7O/SqTVcywyTNxd2xp
```

Comment:
This goes back to an earlier discussion we had I think: Do you know what defines the best feature of all possible features within the search radius? Rather the distance or rather the direction?

Reply:
Yes, I remember. In the meantime I read a bit in the trackpy documentation and in the paper about the Crocker-Grier algorithm.

As I understand it:
As long as we set the method in `tobac ` to _random_, the distance to the original position is relevant. The velocity in successive frames is assumed to be uncorrelated.
If we change the keyword to _predict_, `tobac` uses the _NearestVelocityPredict_ class of `trackpy` and guesses a new position of the cells based on the velocity in the last frame. The distance from this guessed position is then determining the linking. 

In this sense, it is in any case a distance that is decisive. But I think it would be a good idea to create a notebook of an idealized case exploring this, also for my own understanding. So I will start working on this.

Reply:
Thank you for this clarification @snilsn ! That helped a lot and how great that you plan to create another notebook to make this also more clear!